### ISSUE TITLE
Performance Request

### ISSUE THREAD

**Issue by Po-Shin** [deepinsight/insightface#570]
Just a quick question, according to the paper, ResNet50 trained on CASIA  got 99.08% on LFW.

May I ask the performance of the following combination on LFW?
1. ResNet50 model with BN-Dropout-FC-BN output, trained on MS1MV2, using cross entropy loss only
2. ResNet50 model with BN-Dropout-FC-BN output, trained on MS1MV2, using cross entropy loss + center loss

Since I got lower acc. after adding center loss.... that would be great if I can confirm that is normal.

Thanks in advance!

**Comment by nttstar** [comment]
I'm not sure if center loss can work normally now, we're not focusing on it for a while.
